.github/workflows/clippy.yml

```diff
@@ -49,13 +49,13 @@ jobs:
         echo "LD_LIBRARY_PATH=${SYSROOT}/lib${LD_LIBRARY_PATH+:${LD_LIBRARY_PATH}}" >> $GITHUB_ENV
 
     - name: Build
-      run: cargo build --features deny-warnings,internal-lints
+      run: cargo build --features deny-warnings,internal-lints,metadata-collector-lint
```

Comment:
Just to verify, this does not _run_ the metadata collection?

Reply:
Correct. It only includes the source code in the build. Running it still requires the `ENABLE_METADATA_COLLECTION` environment value to be set to `1`.

The following line changes `run: cargo test --features deny-warnings,internal-lints,metadata-collector-lint` will also not run the collection, as the test `dogfood::run_metadata_collection_lint()` is marked as *ignored* :upside_down_face:.

This will therefore basically just ensure that it compiles, which is already a strong insurance in Rust. ^^